### PR TITLE
Onboard b2b organization and role management APIs

### PR DESCRIPTION
**Pull request by sadilchamishka**
Additional improvements made.

- Previously the path is marked as empty as the organization-management APIs were exposed  with the servlet map to `/api/server/v1/organizations`. But as shown in the right hand side of the below image, the `organizations` path is added as with this effort the APIs is exposed with the servlet which expose the server APIs by default `/api/server/v1`

Untitled Diagram-Page-3 (1)

- The above change is done for the role-management API also.

- The package name was changed from `org.wso2.identity.rest.api.organization.management.v1` to `org.wso2.identity.api.server.organization.management.v1` as most of the modules in this repo adhere to that pattern. The same change was done for the role-management API related module.

### Related Issues
- https://github.com/wso2/product-is/issues/16359

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/pom.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```
Comment (ShanChathusanda93) [inline components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/pom.xml:3]: Update the year range. Also please check the license header.

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/pom.xml]: Fixed issues in all the places - https://github.com/wso2/identity-api-server/pull/471/commits/716afd37c24161bdb757d5e2ddea5a111e753ece

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    public OrganizationManager getOrganizationManager() {
+
+        return OrganizationManagementServiceHolder.getInstance().organizationManager;
+    }
+
+    /**
+     * Set OrganizationManager OSGi service.
+     *
+     * @param organizationManager IdentityProviderManager.
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java:76]:
```suggestion
     * @param organizationManager OrganizationManager.
```

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+    public static OrganizationManagementServiceHolder getInstance() {
+
+        return instance;
+    }
+
+    /**
+     * Get OrgApplicationManager OSGi service.
+     *
+     * @return OrgApplicationManager
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java:46]:
```suggestion
     * @return OrgApplicationManager.
```

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    public void setOrgApplicationManager(OrgApplicationManager orgApplicationManager) {
+
+        OrganizationManagementServiceHolder.getInstance().orgApplicationManager = orgApplicationManager;
+    }
+
+    /**
+     * Get OrganizationManager OSGi service.
+     *
+     * @return OrganizationManager
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java:66]:
```suggestion
     * @return OrganizationManager.
```

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java (hunk 3 above)
Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java]: Fixed by - https://github.com/wso2/identity-api-server/pull/471/commits/877411321070f20308eb81238468120b3bace6db

File: components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java (hunk 4 above)
Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/management/common/OrganizationManagementServiceHolder.java]: Fixed by - https://github.com/wso2/identity-api-server/pull/471/commits/877411321070f20308eb81238468120b3bace6db

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**